bin/update_labels.rb

```diff
@@ -7,14 +7,16 @@
 require 'optimist'
 
 opts = Optimist.options do
-  opt :repo, "The repo to update. If not passed, will try all repos in config/repos.yml", :type => :strings
+  opt :repo, "The repo to update. If not passed, will try all repos in config/labels.yml", :type => :strings
   opt :dry_run, "", :default => false
 end
+opts[:repo] ||= ManageIQ::Release::Labels.all.keys
 
 ManageIQ::Release.each_repo(opts[:repo]) do |repo|
-  expected_labels = ManageIQ::Release::Labels[repo.name]
+  expected_labels = ManageIQ::Release::Labels[repo.github_repo]
```

Comment:
why the change from `repo.name` to `repo.github_repo`?

Reply:
Previously the name was ambiguous (as it didn't include the org name), but when I introduced the org concept then the org/name form of the name became important.  This is also why the labels.yml now full qualifies the name.  Additionally, we have a repo that is not in the ManageIQ org (the ansible one), and so, I need the qualifier anyway.